src/sandbox_server.cr

```diff
@@ -82,6 +82,16 @@ module Mint
       artifacts =
         TypeChecker.check(ast)
 
+      runtime =
```

Comment:
Is it better to load the runtime once and just store it in memory? I imagine in a sandbox server restarts and loading of the runtime happens quite often

Reply:
yup, it should be enough to load it just once within the `initialize`.